packages/office-ui-fabric-react/src/components/Modal/Modal.scss

```diff
@@ -1,51 +0,0 @@
-@import '../../common/common';
-
-// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
-
-//
-// Office UI Fabric
-// --------------------------------------------------
-// Modal styles
-
-:export { duration: $ms-animation-duration-2 }
-
-.root {
-  background-color: transparent;
-  position: fixed;
-  height: 100%;
-  width:  100%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  opacity: 0;
-  pointer-events: none;
-  transition: opacity $ms-animation-duration-2;
-
-  :global(.ms-Button.ms-Button--compound) {
-    display: block;
-    @include ms-margin-left(0);
-  }
```

Comment:
Anybody know what this would have been used for? The only thing I can think of is if someone passes button as a child to Modal, but this seems out of the area of concern for Modal.

Reply:
I am pretty sure this can be removed.

Reply:
This likely predates when the css removed margins from components.